src/resources/commandTemplates/templates.ActionCommand.ts

```diff
@@ -639,7 +647,7 @@ export default class ActionCommand extends ResponsiveSlashCommandSubcommandBuild
       .addStringOption(
         new SlashCommandStringOption()
           .setName('duration')
-          .setDescription('Duration of the timeout, if the action is a timeout')
+          .setDescription('Duration of the timeout, if the action is a timeout, or duration to delete past messages for, if the action is a ban')
```

Comment:
The description needs to be fewer than 100 characters, so maybe remove the `if the action is a __` parts?